services/wallet/market/market.go

```diff
@@ -23,12 +23,21 @@ type DataPoint struct {
 	UpdatedAt int64
 }
 
+type MarketValuesSnapshot struct {
+	MarketValues thirdparty.TokenMarketValues
+	UpdatedAt    int64
+}
```

Comment:
Here we're defining a struct that's responsible for time-stamping some `TokenMarketValues`. The time-stamp is useful for determining whether the data is stale.